ncm-metaconfig/src/test/perl/service-logstash.t

```diff
@@ -16,5 +16,9 @@ my $u50 = Test::Quattor::TextRender::Metaconfig->new(
         version => '5.0',
         )->test();
 
+my $u70 = Test::Quattor::TextRender::Metaconfig->new(
+        service => 'logstash',
+        version => '7.0',
+        )->test();
```

Comment:
@stdweird I'm pretty sure the tests for the logstash/7.0 profile don't run. I hoped this would be enough to get them to run but alas. It now says: `Parse errors: No plan found in TAP output`. Any hints on how to proceeded would be highly appreciated.

Reply:
It does:

```
2021-01-27T19:58:10.8531442Z ok 1446 - testspath /quattor_test/ncm-metaconfig/src/main/metaconfig/logstash/7.0/tests exists
2021-01-27T19:58:10.8532384Z ok 1447 - profilespath /quattor_test/ncm-metaconfig/src/main/metaconfig/logstash/7.0/tests/profiles exists
2021-01-27T19:58:10.8533349Z ok 1448 - regexpspath /quattor_test/ncm-metaconfig/src/main/metaconfig/logstash/7.0/tests/regexps exists
2021-01-27T19:58:10.8534025Z ok 1449 - Found regexps
2021-01-27T19:58:10.8534762Z ok 1450 - relpath /quattor_test/ncm-metaconfig/src/main/metaconfig/logstash/7.0/tests/profiles exists and is directory
2021-01-27T19:58:10.8536418Z ok 1451 - panpath /quattor_test/ncm-metaconfig/src/main/metaconfig/logstash/7.0/tests/profiles exists and is directory
2021-01-27T19:58:10.8540596Z ok 1452 - No invalid pan templates
2021-01-27T19:58:10.8541078Z ok 1453 - Found profiles
2021-01-27T19:58:10.8541562Z ok 1454 - All regexps have matching profile
2021-01-27T19:58:10.8541986Z Dubious, test returned 1 (wstat 256, 0x100)
2021-01-27T19:58:10.8542355Z All 1454 subtests passed 
```

So it finds them but doesn't do anything?

Reply:
ok, found it. You have to keep scrolling a lot until you actually see an error.